lib/main.dart

```diff
@@ -1,9 +1,9 @@
 import 'package:flutter/material.dart';
+import 'package:restaurant_bucket_list/searchscreen.dart';
```

Comment:
This file needs general refactor.  
- Split up MyApp and MyHomePage.
- Clean up styling on MyHomePage